src/sentry/testutils/cases.py

```diff
@@ -1129,19 +1130,19 @@ def _push_metric(cls, session, type, key: SessionMRI, tags, value):
 
         def metric_id(key: SessionMRI):
```

Comment:
The type here says `SessionMRI` but the use case is perfomance, which one should be changed?